### PR TITLE
NOISSUE - Change import name aliases

### PR DESCRIPTION
**Pull request by WashingtonKK**
Pull request title should be `MF-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to.
Please review the [CONTRIBUTING.md](https://github.com/mainflux/mainflux/blob/master/CONTRIBUTING.md) file for detailed contributing guidelines.

### What does this do?
Currently, in main.go files we have Go package import aliases that are not according to Go recommended practices. This PR changes the aliases of the imports to correspond to go's recommended practices.

### Which issue(s) does this PR fix/relate to?
No issue

### List any changes that modify/break current functionality
N/A

### Have you included tests for your changes?
N/A

### CODE REVIEW

**Review by SammyOina — COMMENTED** [review]
Check for instances where an alias is used but not required and rebase branch to sync with master

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
also change in other services for example `bootstrap/postgres/setup_test.go`

File: cmd/influxdb-writer/main.go [hunk 1]
```diff
@@ -16,8 +16,8 @@ import (
 	consumerTracing "github.com/mainflux/mainflux/consumers/tracing"
 	"github.com/mainflux/mainflux/consumers/writers/api"
 	"github.com/mainflux/mainflux/consumers/writers/influxdb"
-	influxDBClient "github.com/mainflux/mainflux/internal/clients/influxdb"
-	jaegerClient "github.com/mainflux/mainflux/internal/clients/jaeger"
+	influxdbclient "github.com/mainflux/mainflux/internal/clients/influxdb"
```
Comment (ianmuchyri) [inline cmd/influxdb-writer/main.go:19]: maintain consistency in the naming. In influxdb-reader it is` influxDBclient`

File: cmd/postgres-writer/main.go [hunk 2]
```diff
@@ -16,10 +16,10 @@ import (
 	"github.com/mainflux/mainflux/consumers"
 	consumerTracing "github.com/mainflux/mainflux/consumers/tracing"
```
Comment (ianmuchyri) [inline cmd/postgres-writer/main.go]: `consumertracing`

File: cmd/influxdb-writer/main.go (hunk 1 above)
Comment (WashingtonKK) [reply cmd/influxdb-writer/main.go:19]: Resolved

File: cmd/postgres-writer/main.go (hunk 2 above)
Comment (WashingtonKK) [reply cmd/postgres-writer/main.go]: Solved

**Comment by rodneyosodo** [comment]
> Check for instances where an alias is used but not required and rebase branch to sync with master

Did you check this? For example in `bootstrap/postgres/setup_test.go` you don't need `dockertest "github.com/ory/dockertest/v3"` as the package name is `dockertest` also in `cli/utils.go`
- `cmd/influxdb-reader/main.go` - influxdb2

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
Fix
- `cmd/lora/main.go` redis
- `consumerTracing`

**Review by arvindh123 — CHANGES_REQUESTED** [review]
In the following file `pgClient` is used, Please change them too
users/groups/postgres/setup_test.go
users/policies/postgres/setup_test.go

**Comment by WashingtonKK** [comment]
@arvindh123 , I have addressed the commenrts

**Review by drasko — DISMISSED** [review]
LGTM

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
Fix https://github.com/WashingtonKK/mainflux/blob/639cf561d0836b3fd3deefb1480717e5f10c1eac/cmd/cassandra-writer/main.go#L18 . This applies to all writers `main.go` files

Also there are some places we don't need import alias for example https://github.com/WashingtonKK/mainflux/blob/639cf561d0836b3fd3deefb1480717e5f10c1eac/consumers/notifiers/api/endpoint.go#L10

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
enabled `--enable goimports --enable gci ` on golangcilint

**Review by mteodor — DISMISSED** [review]

**Review by drasko — DISMISSED** [review]
LGTM

File: cmd/lora/main.go [hunk 3]
```diff
@@ -11,7 +11,7 @@ import (
 	"os"
 	"time"
 
-	mqttPaho "github.com/eclipse/paho.mqtt.golang"
+	mqttpaho "github.com/eclipse/paho.mqtt.golang"
 	r "github.com/go-redis/redis/v8"
```
Comment (ianmuchyri) [inline cmd/lora/main.go]: the alias "r"  is very vague

Comment (ianmuchyri) [reply cmd/lora/main.go]: same case in `cmd/opcua/main.go`

**Comment by ianmuchyri** [comment]
Check this in `users/main.go` https://github.com/mainflux/mainflux/blob/2b78902e01709e4eaac0ae9a1e61e6bfafcb38f7/cmd/users/main.go#L48C1-L48C1

**Comment by ianmuchyri** [comment]
https://github.com/mainflux/mainflux/blob/2b78902e01709e4eaac0ae9a1e61e6bfafcb38f7/consumers/writers/cassandra/consumer_test.go#L14

**Comment by ianmuchyri** [comment]
Is the underscore in these necessary: https://github.com/mainflux/mainflux/blob/2b78902e01709e4eaac0ae9a1e61e6bfafcb38f7/pkg/messaging/mqtt/pubsub_test.go#L15  https://github.com/mainflux/mainflux/blob/2b78902e01709e4eaac0ae9a1e61e6bfafcb38f7/pkg/messaging/mqtt/setup_test.go#L18

**Comment by ianmuchyri** [comment]
These aliases are not necessary: https://github.com/mainflux/mainflux/blob/2b78902e01709e4eaac0ae9a1e61e6bfafcb38f7/pkg/messaging/message.pb.go#L13C1-L17C2

**Comment by ianmuchyri** [comment]
Remove unnecessary aliases: https://github.com/mainflux/mainflux/blob/2b78902e01709e4eaac0ae9a1e61e6bfafcb38f7/users/groups/service_test.go#L7C1-L8C11

File: bootstrap/postgres/setup_test.go [hunk 4]
```diff
@@ -9,8 +9,8 @@ import (
 	"testing"
 
 	"github.com/jmoiron/sqlx"
-	bootstrapRepo "github.com/mainflux/mainflux/bootstrap/postgres"
-	pgClient "github.com/mainflux/mainflux/internal/clients/postgres"
+	bootstraprepo "github.com/mainflux/mainflux/bootstrap/postgres"
+	pgclient "github.com/mainflux/mainflux/internal/clients/postgres"
```
Comment (SammyOina) [inline bootstrap/postgres/setup_test.go]: only one of these requires an alias, see other instances in other files where an alias is used but not required

File: certs/postgres/setup_test.go [hunk 5]
```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/jmoiron/sqlx"
 	"github.com/mainflux/mainflux/certs/postgres"
-	pgClient "github.com/mainflux/mainflux/internal/clients/postgres"
+	pgclient "github.com/mainflux/mainflux/internal/clients/postgres"
 	"github.com/mainflux/mainflux/logger"
 	dockertest "github.com/ory/dockertest/v3"
```
Comment (SammyOina) [inline certs/postgres/setup_test.go]: doesn't require an alias

File: cmd/bootstrap/main.go [hunk 6]
```diff
@@ -14,15 +14,15 @@ import (
...
 	rediscons "github.com/mainflux/mainflux/bootstrap/redis/consumer"
 	redisprod "github.com/mainflux/mainflux/bootstrap/redis/producer"
 	"github.com/mainflux/mainflux/bootstrap/tracing"
 	"github.com/mainflux/mainflux/internal"
-	authClient "github.com/mainflux/mainflux/internal/clients/grpc/auth"
-	jaegerClient "github.com/mainflux/mainflux/internal/clients/jaeger"
-	pgClient "github.com/mainflux/mainflux/internal/clients/postgres"
-	redisClient "github.com/mainflux/mainflux/internal/clients/redis"
+	authclient "github.com/mainflux/mainflux/internal/clients/grpc/auth"
+	jaegerclient "github.com/mainflux/mainflux/internal/clients/jaeger"
```
Comment (SammyOina) [inline cmd/bootstrap/main.go]: are these aliases required

File: cmd/bootstrap/main.go [hunk 7]
```diff
@@ -14,15 +14,15 @@ import (
...
+	pgclient "github.com/mainflux/mainflux/internal/clients/postgres"
+	redisclient "github.com/mainflux/mainflux/internal/clients/redis"
```
Comment (SammyOina) [inline cmd/bootstrap/main.go:27]: same for this

**Review by SammyOina — CHANGES_REQUESTED** [review]
check for other instances where an alias is used but not required

File: certs/postgres/setup_test.go (hunk 5 above)
Comment (WashingtonKK) [reply certs/postgres/setup_test.go]: It does, we have:
`	"github.com/mainflux/mainflux/certs/postgres" `
and
`	pgClient "github.com/mainflux/mainflux/internal/clients/postgres"`

File: cmd/bootstrap/main.go (hunk 7 above)
Comment (WashingtonKK) [reply cmd/bootstrap/main.go:27]: pgclient and redis client are needed

File: bootstrap/postgres/setup_test.go [hunk 8]
```diff
@@ -9,8 +9,8 @@ import (
 	"testing"
 
 	"github.com/jmoiron/sqlx"
-	bootstrapRepo "github.com/mainflux/mainflux/bootstrap/postgres"
-	pgClient "github.com/mainflux/mainflux/internal/clients/postgres"
+	"github.com/mainflux/mainflux/bootstrap/postgres"
+	pgclient "github.com/mainflux/mainflux/internal/clients/postgres"
 	"github.com/mainflux/mainflux/logger"
 
 	dockertest "github.com/ory/dockertest/v3"
```
Comment (rodneyosodo) [inline bootstrap/postgres/setup_test.go]:
```go
dockertest "github.com/ory/dockertest/v3"
```
This is not needed as the package is `dockertest`. This applies to all occurrences

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

**Review by SammyOina — CHANGES_REQUESTED** [review]
fix https://github.com/mainflux/mainflux/blob/57d47fea66193b762f6ac68db757d5fc26d7bc82/pkg/ulid/ulid.go#L14 also check for more using golangci-lint

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
Fix https://github.com/WashingtonKK/mainflux/blob/301966046ff8f01040d5a9dfec59911ec981f113/cmd/lora/main.go#L15 and others. It doesn't need import alias

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
Remove unnecessary alias for example:
- `prettyjson "github.com/hokaccha/go-prettyjson"`
- `rediscons "github.com/mainflux/mainflux/bootstrap/redis/consumer"` main.go
- `redisprod "github.com/mainflux/mainflux/bootstrap/redis/producer"` main.go
- `adapter "github.com/mainflux/mainflux/http"` main.go
- `r "github.com/go-redis/redis/v8"` main.go
- `ws "github.com/mainflux/mproxy/pkg/websocket"` main.go
- `redisv8 "github.com/go-redis/redis/v8"` main.go
- `mfsmpp "github.com/mainflux/mainflux/consumers/notifiers/smpp"` main.go
- `twapi "github.com/mainflux/mainflux/twins/api/http"` main.go
- `twmongodb "github.com/mainflux/mainflux/twins/mongodb"` main.go
- `adapter "github.com/mainflux/mainflux/ws"` main.go
- `mux "github.com/plgd-dev/go-coap/v2/mux"`
- `httpapi "github.com/mainflux/mainflux/consumers/notifiers/api"`
- `mfjson "github.com/mainflux/mainflux/pkg/transformers/json"` - consumers/tracing/consumers.go

You can use this to check https://github.com/alexkohler/unimport but it isn't extensive

**Review by dborovcanin — COMMENTED** [review]
This looks good, but it affects a lot of code and will cause conflicts with potentially more important and complex PRs, so let's keep it open until Friday when we'll merge other large PRs.

File: .semaphore/semaphore.yml [hunk 9]
```diff
@@ -101,7 +101,7 @@ blocks:
           commands:
             - cd users
             - cache restore linter
-            - './bin/golangci-lint run'
+            - './bin/golangci-lint run --no-config --disable-all --enable goimports,gofumpt,gci,gosimple,errcheck,govet,unused,goconst,godot,gofumpt --timeout 3m'
```
Comment (dborovcanin) [inline .semaphore/semaphore.yml]: We have added the CI linter config to the root, so no need for these changes.

File: mqtt/handler.go [hunk 10]
```diff
@@ -64,7 +63,8 @@ type handler struct {
 
 // NewHandler creates new Handler entity.
 func NewHandler(publishers []messaging.Publisher, es redis.EventStore,
-	logger logger.Logger, auth policies.AuthServiceClient) session.Handler {
+	logger logger.Logger, auth policies.AuthServiceClient,
+) session.Handler {
```
Comment (dborovcanin) [inline mqtt/handler.go]: Is this change necessary (because of the linter)?

File: scripts/ci.sh [hunk 11]
```diff
@@ -85,7 +85,7 @@ setup() {
 
 run_test() {
     echo "Running lint..."
-    golangci-lint run
+    golangci-lint run --no-config --disable-all --enable gosimple --enable goimports --enable gci --enable errcheck --enable govet --enable gofumpt --enable unused --enable goconst --enable godot --timeout 3m
```
Comment (dborovcanin) [inline scripts/ci.sh]: The same, we have config now.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: .semaphore/semaphore.yml (hunk 9 above)
Comment (WashingtonKK) [reply .semaphore/semaphore.yml]: I don't understand, when we add the linter to root we did not enable the different options

Comment (WashingtonKK) [reply .semaphore/semaphore.yml]: I only added gofumpt to check for fomatting.

File: mqtt/handler.go (hunk 10 above)
Comment (WashingtonKK) [reply mqtt/handler.go]: It is not necessary

File: scripts/ci.sh (hunk 11 above)
Comment (WashingtonKK) [reply scripts/ci.sh]: Understood

File: .semaphore/semaphore.yml (hunk 9 above)
Comment (WashingtonKK) [reply .semaphore/semaphore.yml]: Understood

File: .semaphore/semaphore.yml [hunk 12]
```diff
@@ -101,7 +101,7 @@ blocks:
           commands:
             - cd users
             - cache restore linter
-            - './bin/golangci-lint run'
+            - './bin/golangci-lint run
```
Comment (dborovcanin) [inline .semaphore/semaphore.yml]: Is this a typo? What's the reason for this change?

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: .semaphore/semaphore.yml (hunk 12 above)
Comment (WashingtonKK) [reply .semaphore/semaphore.yml]: Yes it is, let me resolve

**Review by dborovcanin — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM
